webpack/stories/docs/tours.md

```diff
@@ -0,0 +1,45 @@
+# Tour
+
+## React Components
+In order to create a tour of your React component, please add a `<YourComponent>.tour.js` file in the component's library
+
+### Example
+```js
+import withTour from '../../common/Tour';
+import YourComponent from '.';
+
+const steps = [
+  {
+    selector: '[data-tut="selector_1"]',
+    content: __('Welcome the new feature !'),
+  },
+  {
+    selector: '[data-tut="selector_2"]',
+    content: __('This is step 2'),
+  },
+  {
+    selector: '[data-tut="selector_3"]',
+    content: __('This is another step'),
+  },
+];
+
+export default withTour(YourComponent, steps, '<unique ID>');
+```
+
+Each selector is a one step of your tour, add a data attribute named `data-tut` on each element which is displayed as a step.
+The `WithTour` HOC adds a special callback named `runTour` which adding this tour to a runnig queue.
+You can call `runTour` on your wrapped component everywhere, for example under `componentDidMount` for instant running, or under a click event.
+
+
+## Rails Views
+Tour can run over rails view content as well!
+Please note that the tour will run instantly on page load.
+
+### Example
+```ruby
+<%= run_tour('<unique ID>', [
```

Comment:
Cool, I didn't test with Ruby yet!  Can you add an example tour for a Rails page?

Reply:
@amirfefer?